src/openfermion/utils/_davidson.py

```diff
@@ -158,6 +162,8 @@ def get_lowest_n(self, n_lowest=1, initial_guess=None, max_iterations=None):
         while (num_iterations < max_iterations and not success):
             (eigen_values, eigen_vectors, mat_eigen_vectors, max_trial_error,
              guess_v, guess_mv) = self._iterate(n_lowest, guess_v, guess_mv)
+            logging.info("Eigenvalues for iteration %d: %s, error is %f.",
```

Comment:
I'm not opposed to it, but have we been using logging anywhere else in the code? Should we start?

Reply:
In [examples/](https://github.com/quantumlib/OpenFermion/blob/master/examples/performance_benchmarks.py), I replaced `print` with `logging.info`, which I believe is a more standard way for tracing.

Though it's not present in examples/ yet, I'm sure it'll be useful:
* in production;
* for debugging (I tried to log it in unit test, it doesn't work yet, not sure why :( ).